tests/test-dirs/signature-help/sh-mix.t

```diff
@@ -0,0 +1,49 @@
+  $ cat >test.ml <<'EOF'
+  > let f x ~lbl_a y ~lbl_b z = ignore (x, lbl_a, lbl_b, z)
+  > let _ = f
+  > EOF
+
+First is the first parameter:
+  $ $MERLIN single signature-help -position 2:11 -filename test <test.ml |
+  > jq '.value.activeParameter'
+  0
+
+  $ cat >test.ml <<'EOF'
+  > let f x ~lbl_a y ~lbl_b z = ignore (x, lbl_a, lbl_b, z)
+  > let _ = f 0 
+  > EOF
+
+FIXME: After we expect the next non-labelled parameter to be active:
```

Comment:
@voodoos 
even simpler case for this would be
```
It can make a third non-labelled parameter active.

  $ $MERLIN single signature-help -position 2:14 <<EOF
  > let f x y z = ignore (x, y, z)
  > let _ = f 0 1
```
result will be activeParameter equal one. As I understand it, the reason is that we search by arguments and when you just put the space after an argument, there are no arguments in typed tree.